src/coreclr/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -547,7 +547,7 @@ internal static void NotifyThreadUnblocked()
              RegisteredWaitHandle registeredWaitHandle
              );
 
-        [Obsolete("ThreadPool.BindHandle(IntPtr) has been deprecated.  Please use ThreadPool.BindHandle(SafeHandle) instead.", false)]
+        [Obsolete("ThreadPool.BindHandle(IntPtr) has been deprecated. Use ThreadPool.BindHandle(SafeHandle) instead.", false)]
```

Comment:
```suggestion
        [Obsolete("ThreadPool.BindHandle(IntPtr) has been deprecated. Use ThreadPool.BindHandle(SafeHandle) instead.")]
```

Reply:
Thanks; I had used a regex, so now I have 2 problems. I did another pass and found 1 more occurrence too.